src/Commands/Org/Site/ListCommand.php

```diff
@@ -26,7 +26,7 @@ class ListCommand extends TerminusCommand implements SiteAwareInterface
      * @field-labels
      *     name: Name
      *     id: ID
-     *     service_level: Service Level
```

Comment:
Are service levels going to be completely removed from the platform by the time this PR is merged? If there are still going to be legacy sites that are still on a service level, then you could retain the `service_level` field, but omit it from `@default-fields`. This would give folks a way to still display their legacy plan service level on request.

If service_level is going to be completely removed from the underlying API, though, then might as well remove it as you have done here.

Reply:
The `plan_name` attribute is present on legacy sites.